Documentation/JSON_Schemas/HeartBeat_Response.json

```diff
@@ -0,0 +1,106 @@
+{
+  "type": "object",
+  "properties": {
+    "user": {
```

Comment:
Yes, token can be removed, but it's a nice method of informing the client if it should act like caesar.